packages/markitdown/src/markitdown/converters/_bing_serp_converter.py

```diff
@@ -79,7 +79,9 @@ def convert(
             slug.extract()
 
         # Parse the algorithmic results
-        _markdownify = _CustomMarkdownify()
+        _markdownify = _CustomMarkdownify(
+            keep_data_uris=kwargs.get("keep_data_uris", False)
+        )
```

Comment:
```suggestion
        _markdownify = _CustomMarkdownify(
            **kwargs
        )
```
By this way, we lay a "foundation" so that we can easily add new options to `_CustomMarkdownify` without changing too much in upstream objects